2016/10/06 09:54:47 [error] [exception.CException] exception 'CException' with message 'Property "SiteController.index" is not defined.' in C:\xampp\htdocs\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\ensbodas\protected\views\layouts\main.php(30): CComponent->__get('index')
#1 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\Yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\Yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\Yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\ensbodas\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#10 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#11 C:\xampp\htdocs\Yii\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#12 C:\xampp\htdocs\ensbodas\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\Yii\framework\web\CWebApplication.php(282): CController->run('')
#18 C:\xampp\htdocs\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#19 C:\xampp\htdocs\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#20 C:\xampp\htdocs\ensbodas\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/ensbodas/
---
2016/10/06 09:58:06 [error] [exception.CException] exception 'CException' with message 'Property "SiteController._id" is not defined.' in C:\xampp\htdocs\Yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\ensbodas\protected\views\layouts\main.php(30): CComponent->__get('_id')
#1 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\Yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\Yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\Yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\ensbodas\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#10 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#11 C:\xampp\htdocs\Yii\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#12 C:\xampp\htdocs\ensbodas\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\Yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\Yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\Yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\Yii\framework\web\CWebApplication.php(282): CController->run('')
#18 C:\xampp\htdocs\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#19 C:\xampp\htdocs\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#20 C:\xampp\htdocs\ensbodas\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/ensbodas/
---
2016/10/06 10:04:20 [error] [php] Object of class CInlineAction could not be converted to string (C:\xampp\htdocs\ensbodas\protected\views\layouts\main.php:31)
Stack trace:
#0 C:\xampp\htdocs\Yii\framework\web\widgets\CContentDecorator.php(76): SiteController->renderFile()
#1 C:\xampp\htdocs\Yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#2 C:\xampp\htdocs\Yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#3 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(206): CContentDecorator->run()
#4 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(300): SiteController->endWidget()
#5 C:\xampp\htdocs\ensbodas\protected\views\layouts\column1.php(6): SiteController->endContent()
#6 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(126): require()
#7 C:\xampp\htdocs\Yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#8 C:\xampp\htdocs\Yii\framework\web\CController.php(784): SiteController->renderFile()
#9 C:\xampp\htdocs\ensbodas\protected\controllers\SiteController.php(32): SiteController->render()
#10 C:\xampp\htdocs\Yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#11 C:\xampp\htdocs\Yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#12 C:\xampp\htdocs\Yii\framework\web\CController.php(286): SiteController->runAction()
#13 C:\xampp\htdocs\Yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#14 C:\xampp\htdocs\Yii\framework\web\CWebApplication.php(282): SiteController->run()
#15 C:\xampp\htdocs\Yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\Yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 C:\xampp\htdocs\ensbodas\index.php(13): CWebApplication->run()
REQUEST_URI=/ensbodas/index.php/site/index
in C:\xampp\htdocs\ensbodas\protected\views\layouts\main.php (31)
in C:\xampp\htdocs\ensbodas\protected\views\layouts\column1.php (6)
in C:\xampp\htdocs\ensbodas\protected\controllers\SiteController.php (32)
2016/10/10 23:30:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] No such file or directory' in /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /Applications/XAMPP/xamppfiles/htdocs/bodasens/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/bodasens/index.php/gii/model/index
HTTP_REFERER=http://localhost/bodasens/index.php/gii/default/index
---
2016/10/10 23:31:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] No such file or directory' in /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /Applications/XAMPP/xamppfiles/htdocs/bodasens/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/bodasens/index.php/gii/model/index
HTTP_REFERER=http://localhost/bodasens/index.php/gii/default/index
---
2016/10/10 23:42:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /Applications/XAMPP/xamppfiles/htdocs/bodasens/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/bodasens/index.php/gii/model/index
HTTP_REFERER=http://localhost/bodasens/index.php/gii/default/index
---
2016/10/10 23:43:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /Applications/XAMPP/xamppfiles/htdocs/bodasens/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/bodasens/index.php/gii/model/index
HTTP_REFERER=http://localhost/bodasens/index.php/gii/default/index
---
2016/10/11 01:18:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php:483
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(270): CController->missingAction('index.php')
#1 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(282): CController->run('index.php')
#2 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contacto/index....')
#3 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/bodasens/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/bodasens/index.php/contacto/index.php
---
2016/10/11 01:19:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php:483
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CController.php(270): CController->missingAction('index.php')
#1 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(282): CController->run('index.php')
#2 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/web/CWebApplication.php(141): CWebApplication->runController('Acercade/index....')
#3 /Applications/XAMPP/xamppfiles/htdocs/Yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#4 /Applications/XAMPP/xamppfiles/htdocs/bodasens/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/bodasens/index.php/Acercade/index.php
---
